### PR TITLE
[Android] Modify README.md contents for ssd-autocapture

### PR DESCRIPTION
**Pull request by macaron10**
[Edit] README.md : Modify contents with detail information about nnstreamer-ssd-autocapture
[Add] screenshot_autocapture-off.jpg, screenshot_autocapture-on.jpg, screenshot_preview.jpg, screenshot_setting.jpg : Add screenshots for README.md

Signed-off-by: Kanggyu.Min <macaron61@gmail.com>

### CODE REVIEW

File: android/example_app/nnstreamer-ssd-autocapture/README.md [hunk 1]
```diff
@@ -1,6 +1,68 @@
...
+
+
+
+## Usage Examples
+
+#### Main screen
+
+- AutoCapture Off / On
+
+  ​                       <img src="D:\Project\SR_Project\nnstreamer-example\android\example_app\nnstreamer-ssd-autocapture\res\screenshot\screenshot_autocapture-off.jpg" alt="image-autocapture-off" style="zoom: 67%;" />                       <img src="D:\Project\SR_Project\nnstreamer-example\android\example_app\nnstreamer-ssd-autocapture\res\screenshot\screenshot_autocapture-on.jpg" alt="image-autocapture-on" style="zoom:67%;" />
```
Comment (anyj0527) [inline android/example_app/nnstreamer-ssd-autocapture/README.md:48]: Please check the urls. Use github based link!

Comment (macaron10) [reply android/example_app/nnstreamer-ssd-autocapture/README.md:48]: Thank you very much! I will try again, after editing the urls.

**Review by Joojihwan — COMMENTED** [review]
LGTM!

**Comment by JeongYeonUk** [comment]
LGTM!

**Comment by Soyeong-k-1230** [comment]
LGTM!!

**Comment by park-sang-hyun** [comment]
LGTM!

**Comment by anyj0527** [comment]
Please make these as a single commit. Refer to Jihoon's comment https://github.com/nnstreamer/nnstreamer-example/pull/217#issuecomment-703166459

**Review by anyj0527 — APPROVED** [review]
LGTM 👍🏻👍🏻

**Pull request merged**